app/assets/javascripts/application.js

```diff
@@ -77,7 +77,7 @@ function onContentLoad(){
   $("#title_action a").addClass('btn').not('.btn-info, .btn-danger').addClass("btn-default");
 
   $("#title_action li a").removeClass("btn btn-default").addClass("la");
-  $("#title_action span").removeClass("btn btn-default").addClass("btn-group");
+  $("#title_action span").not('.glyphicon').removeClass("btn btn-default").addClass("btn-group");
```

Comment:
Why are we using js to change these classes instead of setting the right classes in the first place?